pkgs/applications/office/super-productivity/default.nix

```diff
@@ -2,16 +2,17 @@
 
 stdenv.mkDerivation rec {
   pname = "super-productivity";
-  version = "9.0.5";
+  version = "10.0.2";
 
   src = fetchurl {
     url = "https://github.com/johannesjo/super-productivity/releases/download/v${version}/superProductivity-${version}.AppImage";
-    sha256 = "sha256-eNAoLcQWnsTDA7sG8i0Ur9BZ+pNt4AK1GOppFCD1ZGg=";
+    hash = "sha256-TPe2wijTzlro9rtESJHPL1phtraSW5kTHfctSTTumGs=";
     name = "${pname}-${version}.AppImage";
   };
 
   appimageContents = appimageTools.extractType2 {
-    inherit pname version src;
+    name = "${pname}-${version}";
+    inherit src;
```

Comment:
Why? Please remove this change.

Reply:
okay, thanks for the help it's the first time i ever contribute to nixpkgs